docs/src/tutorials/creating_geometry.md

```diff
@@ -0,0 +1,251 @@
+# Creating Geometry
+
+````@example creating_geometry
+import GeoInterface as GI
+import GeometryOps as GO
+import GeoFormatTypes as GFT
+import CoordinateTransformations
+import Proj
+using CairoMakie
+using GeoMakie
+using GeoJSON
+````
+
+The first thing we need to do is decide which Coordinate Reference System (CRS) we will be working in. Here, we start with the most common geographic CRS (i.e. coordiantes of latitude and longitude), [WGS84](https://epsg.io/4326).
+
+````@example creating_geometry
+crs = GFT.EPSG(4326)
+````
+
+Let's start by making a single `Point` with CRS info included.
+
+````@example creating_geometry
+point = GI.Point(0, 0; crs)
+````
+
+Now, let's plot our point.
+
+````@example creating_geometry
+fig, ax, plt = plot(point)
+````
+
+Let's create a set of points, and have a bit more fun with plotting.
+
+````@example creating_geometry
+x = [-5, 0, 5, 0]
+y = [0, -5, 0, 5]
+points = GI.Point.(zip(x,y); crs)
+plot!(ax, points; marker = '✈', markersize = 30)
+fig
+````
+
+Points can be combined into a single `MultiPoint` geometry.
+
+````@example creating_geometry
+x = [-5, -5, 5, 5]
+y = [-5, 5, 5, -5]
+multipoint = GI.MultiPoint(GI.Point.(zip(x, y); crs))
+plot!(ax, multipoint, marker = '☁', markersize = 30)
+fig
+````
+
+Let's create a line between two points.
+
+````@example creating_geometry
+p1 = GI.Point.(-5, 0; crs)
```

Comment:
Like now these have crs too... Even though they're going in the Line strong.

Reply:
OK, ok... we'll remove the crs. It seems like higher-level constructors should strip lower level CRS info.. and I'm guessing there is a good reason that this wasn't implemented (reallocation?).

Reply:
That would require a second allocation (bad). Also confusing Because GI geoms just wrap whatever they are given as-is, including points from other packages that they don't know how to modify.

(Im making this point like this because I have thought about it a lot, like while writing the code for GI geoms...)